src/FunctionalTest/Smo/ScriptingTests/PartitionFunction_SmoTestSuite.cs

```diff
@@ -60,6 +62,32 @@ public void SmoDropIfExists_PartitionFunction_Sql16AndAfterOnPrem()
                 });
         }
 
+        /// <summary>
+        /// Tests create script of a partition function handling DBNull range values correctly
+        /// </summary>
+        [TestMethod]
+        public void ScriptCreateWithDbNullRangeCorrectly_PartitionFunction()
+        {
+            ExecuteFromDbPool(db =>
+            {
+                // Create a Partition Function using SMO
+                var partitionFunction = new _SMO.PartitionFunction(db, "PF_NULL_RANGE_TEST");
```

Comment:
Can't you just use an object initializer for those 2 properties

```csharp
new _SMO.PartitionFunction(db, "PF_NULL_RANGE_TEST") { RangeType = .. ; RangeValues = new object[] { ... } };
```

Reply:
Sure that's possible. I tried to be consistent with the rest of the test suite.